src/components/shared/page.tsx

```diff
@@ -22,15 +22,28 @@ import { insertAdPlaceholders } from 'ads';
 // ----- Components ----- //
 
 const PageStyles = css`
-    ${fontFace("Guardian Egyptian Web", new None, "/public/fonts/egyptiantext-regular.otf")}
-    ${fontFace("Guardian Egyptian Web", new Some(500), "/public/fonts/egyptiantext-medium.otf")}
-    ${fontFace("Guardian Egyptian Web", new Some("bold"), "/public/fonts/egyptiantext-medium.otf")}
-    ${fontFace("Guardian Text Sans Web", new None, "/public/fonts/GuardianSansWeb-Regular.ttf")}
-    ${fontFace("Guardian Headline", new Some(200), "/public/fonts/GHGuardianHeadline-Light.ttf")}
-    ${fontFace("Guardian Headline", new Some(400), "/public/fonts/GHGuardianHeadline-Regular.ttf")}
-    ${fontFace("Guardian Headline", new Some(500), "/public/fonts/GHGuardianHeadline-Medium.ttf")}
-    ${fontFace("Guardian Headline", new Some(700), "/public/fonts/GHGuardianHeadline-Bold.ttf")}
-    ${fontFace("Guardian Icons", new None, "/public/fonts/icons.otf")}
+    ${fontFace("Guardian Egyptian Web", new None, new None, "/public/fonts/GuardianTextEgyptian-Reg.ttf")}
+    ${fontFace("Guardian Egyptian Web", new None, new Some("italic"), "/public/fonts/GuardianTextEgyptian-RegItalic.ttf")}
+    ${fontFace("Guardian Egyptian Web", new Some(500), new None, "/public/fonts/GuardianTextEgyptian-Bold.ttf")}
+    ${fontFace("Guardian Egyptian Web", new Some("bold"), new None, "/public/fonts/GuardianTextEgyptian-Bold.ttf")}
+    ${fontFace("Guardian Egyptian Web", new Some("bold"), new Some("italic"), "/public/fonts/GuardianTextEgyptian-BoldItalic.ttf")}
```

Comment:
Is this not `Guardian Text Egyptian Web`?

Reply:
Aren’t `Guardian Egyptian Web` headline fonts (formerly Egyptian Headline, now Guardian Headline)? (easy way to check would be their use in headlines). If yes – they should be mapped to `GH Guardian Headline` family. If they are indeed Guardian Text Egyptians, though, they were/are just misnamed.

If that’s confusing, just grab me in person!

Reply:
I think it's the second case, i.e. they're `Guardian Text Egyptian` and just misnamed, because the filenames are e.g. `GuardianTextEgyptian-Bold.ttf`.